ota-plus-web/app/reactapp/src/stores/UserStore.js

```diff
@@ -42,10 +43,15 @@ export default class UserStore {
     resetAsync(this.contractsAcceptAsync);
   }
 
-  getNamespaces = async () => {
-    const response = await axios.get(API_USER_NAMESPACES);
+  getOrganizations = async () => {
+    const response = await axios.get(API_USER_ORGANIZATIONS);
     const { data } = await response;
-    this.userNamespace = data.length ? data[0] : '';
+    this.userOrganizationName = data.find(organization => organization.landing).name;
+    if (!this.userOrganizationName) {
```

Comment:
@houcros are you doing this on BE too?

Reply:
Yes, this should be merged with https://github.com/advancedtelematic/ota-plus-user-profile/pull/84.

Reply:
(But then there would be no need to use the initials here.)